resources/views/admin_partials/navbar.blade.php

```diff
@@ -1,7 +1,7 @@
 <nav class="navbar navbar-static-top" role="navigation">
```

Comment:
path to this file should be `resource/views/layouts/admin/partials/navbar.blade.php`